### ISSUE TITLE
[Regression] [macOS] [Arm] Compiling fails after #12620 

### ISSUE THREAD

**Issue by shinra-electric** [RPCS3/rpcs3#12774]
### Quick summary

Compilation on Arm fails (Xcode 14.0.1)

### Details

Compilation on Arm fails (Xcode 14.0.1) with the following warning: 

```
/Users/XXX/Downloads/rpcs3/rpcs3/Emu/RSX/VK/vkutils/device.cpp:154:15: error: no member named 'semaphoreSupportStyle' in 'MVKConfiguration'
                        mvk_config.semaphoreSupportStyle = MVKVkSemaphoreSupportStyle::MVK_CONFIG_VK_SEMAPHORE_SUPPORT_STYLE_SINGLE_QUEUE;
                        ~~~~~~~~~~ ^
/Users/XXX/Downloads/rpcs3/rpcs3/Emu/RSX/VK/vkutils/device.cpp:154:39: error: use of undeclared identifier 'MVKVkSemaphoreSupportStyle'
                        mvk_config.semaphoreSupportStyle = MVKVkSemaphoreSupportStyle::MVK_CONFIG_VK_SEMAPHORE_SUPPORT_STYLE_SINGLE_QUEUE;
```

A change was made to this line in #12620 so I suspect this is the cause. 

### Build with regression

#12620

### Attach two log files

N/A

### Attach capture files for visual issues

N/A

### System configuration

M1

### Other details

Perhaps @nastys can have a look?

**Comment by shinra-electric** [comment]
I just came across [this](https://github.com/KhronosGroup/MoltenVK/pull/1738) on the MoltenVK GitHub, and it may be related.

**Comment by tellowkrinkle** [comment]
That is indeed related.  It should now default to either SingleQueue or MTLEvents (both of which perform well), without needing any special config changes.  The correct solution is to remove that line, as it's no longer needed.
